src/HonzaBotner/appsettings.CvutFit.json

```diff
@@ -20,7 +20,10 @@
         "VerificationMessageId": 686745124885364770,
         "VerificationChannelId": 507515506073403402,
         "VerificationEmojiName": "✅",
-        "StaffVerificationEmojiName": "👑"
+        "StaffVerificationEmojiName": "👑",
+        "HornyJailRoleId": 768941177872318484,
+        "HornyJailChannelId": 768941687187177472,
+        "HornyJailFilePath": "../../docs/hornyjail.mp4"
```

Comment:
Have you verified that the file is added to the image when building in heroku?

Reply:
No.

Reply:
Is there any good way how to add resources to a build?

Reply:
Why should we upload new video every time someone is added if it is not personalized. I think discord allows to show video just by mentioning link to the video.

Reply:
This is good point. And we wouldnt have to worry about the building issue.

Reply:
@pokorj54 I am not sure this works with videos too. It works with images/gifs. We might personalize it in the future. For files, it doesn't matter if it uploads files from an external URL or from a local file.

Reply:
@tenhobi I am quite sure it works because I often use it if I don't want to download manually the video and then upload it, I just copy the link and paste it. I don't know how this link is created, but it is used in Wormhole to not upload files a second time, so Cauchy will know how it is implemented.

Reply:
@ostorc we should move the file to the `hb.Discord.Services` project tho, to some resource directory. That way we can Include the directory in the build. (`ItemGroup` > `Folder` in .csproj).